gsa/src/web/pages/permissions/multipledialog.js

```diff
@@ -237,10 +245,15 @@ MultiplePermissionDialog.propTypes = {
   userId: PropTypes.id,
   users: PropTypes.array,
   visible: PropTypes.bool,
+  onChange: PropTypes.func,
   onClose: PropTypes.func.isRequired,
   onSave: PropTypes.func.isRequired,
 };
 
+MultiplePermissionDialog.contextTypes = {
+  capabilities: PropTypes.capabilities.isRequired,
+};
+
```

Comment:
Could you remove the MultiplePermissionDialog.contextTypes = { lines? Not sure why it did get added again. I've removed it in one of my last pull requests because it is obsolete and unnecessary here.